qutebrowser/browser/qutescheme.py

```diff
@@ -159,6 +161,35 @@ def qute_bookmarks(_url):
     return 'text/html', html
 
 
+@add_handler('history')
+def qute_history(_url):
+    """Handler for qute:history. Display browsing history."""
+    history = reversed(objreg.get('web-history').history_dict.values())
+
+    fmt = config.get('completion', 'timestamp-format')
+    if fmt is None:
+        def fmt_atime(atime):
+            return ''
+    else:
```

Comment:
Why define two functions at all, instead of just having one function and doing the `if fmt is None:` check in there?